### DISCUSSION TITLE
The problem with the Electrum wallet

### DISCUSSION THREAD

**Discussion by Usovsergey** [community/community#78979] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Issues

### Body

Hello.
Yesterday I made 2 payments from my Electrum wallet. 
Payments have not been executed for a long time. 
Today I found out that there were errors on them and tried to cancel both payments. 
We managed to put one payment on the refund, but the operation has not been performed for a long time. 
In the second payment, the system reports an error. 
Please help me get my money back or inform Electrum wallet support.

Operations:
No. 1. a139d66ad5f88c6829d2422d53dd68125a085af931811aa722918ff5df9a58e6
No. 2. 773122cc9dd008ae52bbae35557a93001140dc6cf4bf7ad77002b24bbc942d5b

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Usovsergey ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by Usovsergey** [comment]
Thank you, I will look for it.
